core-data/selectors.js

```diff
@@ -61,13 +61,15 @@ export function getMedia( state, id ) {
 /**
  * Returns the Post Type object by slug.
  *
- * @param {Object} state Data state.
- * @param {number} slug  Post Type slug.
+ * @param {Object} state       State tree
+ * @param {string} kind        Model kind.
+ * @param {string} name        Model name.
+ * @param {number} primaryKey  Record's Primary key
  *
- * @return {Object?}     Post Type object.
+ * @return {Object?} Record.
  */
-export function getPostType( state, slug ) {
```

Comment:
I assume we have some using the existing selector to be updated (work-in-progress) ?

Reply:
This PR now generates dynamically the selector in core-data/model-selectors.js. So everything keeps working as before :)